ui/src/components/admin/helpers/googleAuth.js

```diff
@@ -0,0 +1,64 @@
+import jwt_decode from 'jwt-decode';
+
+const GOOGLE_AUTH_KEY = 'HCMI_GOOGLE_AUTH';
+
+/*
+  AuthResponse object: https://developers.google.com/identity/sign-in/web/reference#gapiauth2authresponse
+  Returned after Google sign-in, used to authorize requests in CMS backend
+ */
+export const getAuth = () => {
```

Comment:
Since the new Google Identity Services Sign-In doesn't keep track of the user's session, just signs the user in and returns an auth code, we store the auth response from the backend (used for authorizing requests) after the user signs in.